### PR TITLE
Add Full Title

### PR DESCRIPTION
**Pull request by phudv-0958**
Add full title

### CODE REVIEW

**Comment by phudv-0958** [comment]
ready

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+  def full_title(page_title = '')
+    base_title = t ".base_title"
+    if page_title.empty?
```
Comment (baynv-0959) [inline app/helpers/application_helper.rb]:
```suggestion
return if page_title.empty?
page_title + " | " + base_title
 end
```

**Comment by phudv-0958** [comment]
ready

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,8 @@
 module ApplicationHelper
+  def full_title page_title = ''
```
Comment (phucmh-1356) [inline app/helpers/application_helper.rb]: dùng "" thay cho '

**Comment by phudv-0958** [comment]
ready

**Pull request merged**